### PR TITLE
feat(tiflow): disable old jobs trigger on lts branch

### PR DESCRIPTION
**Pull request by purelind**
disable old jobs trigger on some LTS branch
* release-6.5
* release-6.1
* release-5.4
* release-5.3

### CODE REVIEW

File: jenkins/jobs/ci2/pingcap/tiflow/cdc_ghpr_integration_test.groovy [hunk 1]
```diff
@@ -25,6 +25,8 @@ pipelineJob('cdc_ghpr_integration_test') {
                     blackListTargetBranches {
                         ghprbBranch { branch('master') }
                         ghprbBranch { branch('^(release-)?7\\.[1-9]\\d*(\\.\\d+)?(\\-.*)?$') }
+                        ghprbBranch { branch('^(release-)?5\\.[3-4]\\d*(\\.\\d+)?(\\-.*)?$') }
+                        ghprbBranch { branch('^(release-)?6\\.[1|5]\\d*(\\.\\d+)?(\\-.*)?$') }
```
Comment (wuhuizuo) [inline jenkins/jobs/ci2/pingcap/tiflow/cdc_ghpr_integration_test.groovy:29]: maybe we should sort them: 5,6,7 ...

File: jenkins/jobs/ci2/pingcap/tiflow/dm_ghpr_compatibility_test.groovy [hunk 2]
```diff
@@ -23,12 +23,13 @@ pipelineJob('dm_ghpr_compatibility_test') {
                     whitelist("")
                     orgslist("pingcap")
                     whiteListTargetBranches {
-                        ghprbBranch { branch('^(release-)?[5]\\.[3-4](\\.\\d+)?(\\-.*)?$') }
                         ghprbBranch { branch('^(release-)?[6]\\.[0-9](\\.\\d+)?(\\-.*)?$') }
                         ghprbBranch { branch('^(release-)?[7]\\.0(\\.\\d+)?(\\-.*)?$') }
```
Comment (wuhuizuo) [inline jenkins/jobs/ci2/pingcap/tiflow/dm_ghpr_compatibility_test.groovy:27]: Also those lines should be deleted.

File: jenkins/jobs/ci2/pingcap/tiflow/dm_ghpr_compatibility_test.groovy [hunk 3]
```diff
@@ -23,12 +23,13 @@ pipelineJob('dm_ghpr_compatibility_test') {
...
                     }
                     blackListTargetBranches {
                         ghprbBranch { branch('master') }
+                        ghprbBranch { branch('^(release-)?5\\.[3-4]\\d*(\\.\\d+)?(\\-.*)?$') }
+                        ghprbBranch { branch('^(release-)?6\\.[1|5]\\d*(\\.\\d+)?(\\-.*)?$') }
```
Comment (wuhuizuo) [inline jenkins/jobs/ci2/pingcap/tiflow/dm_ghpr_compatibility_test.groovy:32]: missing line for `7`?

Comment (purelind) [reply jenkins/jobs/ci2/pingcap/tiflow/dm_ghpr_compatibility_test.groovy:32]: It is not necessary when`whiteListTargetBranches` has already been configured.

**Pull request merged**